guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,6 +4,19 @@
 [id="katello-headline-features"]
 == Headline Features
 
+* New Alternate Content Sources improvments
+- Alternate Content Sources support for RHUI types.
+- Alternate Content Sources support for bulk refresh and delete.
+- Alternate Content Sources has been moved out of labs into content on the vertial nav bar.
+- Content Credentials has been updated to support Alternate Content Sources.
+* New Host UI improvments
+- Clicking a Content Host on the legacy ui, now takes you to the new host UI page.
+- Applicable errata now show as an option on the ErrataOverviewCard
+* New Inter Server Sync improvments
+- Syncable imports can now sync from a URL instead of a file path
+- Yum repository now has support for syncable formats
```

Comment:
```suggestion
** Alternate Content Sources support for RHUI types.
** Alternate Content Sources support for bulk refresh and delete.
** Alternate Content Sources has been moved out of labs into content on the vertial nav bar.
** Content Credentials has been updated to support Alternate Content Sources.
* New Host UI improvments
** Clicking a Content Host on the legacy ui, now takes you to the new host UI page.
** Applicable errata now show as an option on the ErrataOverviewCard
* New Inter Server Sync improvments
** Syncable imports can now sync from a URL instead of a file path
** Yum repository now has support for syncable formats
```

Reply:
Fixed.